.github/workflows/ci.yml

```diff
@@ -1,16 +1,38 @@
 name: CI
 
 on:
+  workflow_dispatch:
   pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
   push:
-    branches: main
+    branches:
+      - main
+      - nightly
   schedule:
-    - cron: "0 18 * * 1,4,6" # 1800 UTC every Monday, Thursday, Saturday
+    - cron: "30 18 * * 1,4,6" # 1830 UTC every Monday, Thursday, Saturday
 
 jobs:
+  debug:
+    name: Debug context
+    runs-on: ubuntu-latest
+    steps:
+      - name: Debug context
+        env:
+          ASSOC: ${{ github.event.pull_request.author_association }}
+        run: |
+          echo author_association = $ASSOC
+
+      
   tests:
     name: Unit tests
-    if: contains(github.event.pull_request.labels.*.name, 'safe to test')
+    if: |
+      contains(github.event.pull_request.labels.*.name, 'safe to test') ||
+      github.event.pull_request.author_association == 'COLLABORATOR' ||
```

Comment:
`COLLABORATOR` means anyone who is explicitly allow-listed in the repo Settings page.

`MEMBER` means anyone who is allow-listed in the organization's Settings page.